examples/pytorch/text-classification/sparseml_utils.py

```diff
@@ -0,0 +1,247 @@
+import collections
+import inspect
+import math
+import os
+from typing import Any, Optional
+
+import numpy
+import torch
+import torch.nn.functional as F
+
+import onnxruntime
+from sparseml.pytorch.optim.manager import ScheduledModifierManager
+from sparseml.pytorch.optim.optimizer import ScheduledOptimizer
+from sparseml.pytorch.utils import ModuleExporter, logger
+from transformers import Trainer
+from transformers.file_utils import RECIPE_NAME, WEIGHTS_NAME
+from transformers.modeling_outputs import SequenceClassifierOutput
+from transformers.models.bert.modeling_bert import BertForSequenceClassification
+
+
+class SparseMLGLUETrainer(Trainer):
```

Comment:
This class shares a lot with SparseMLQATrainer that could be abstracted out and reused. Do you think we can put them into sth like SparseMLTrainer?